Sources/Receipt/ReceiptTransactionTableViewCell.swift

```diff
@@ -130,18 +130,18 @@ extension ReceiptTransactionTableViewCell {
         guard let receipt = receipt else {
```

Comment:
add a check in here let receipt = receipt, let entry = receipt.entry?.rawValue, let createOn = receipt.createdOn else {
}